Extensions/PlatformBehavior/platformtools.ts

```diff
@@ -0,0 +1,18 @@
+namespace gdjs {
+  export namespace platform {
```

Comment:
Let's put this inside the `evtTools` namespace (gdjs.evtTools.platform).

The `gdjs` namespace is fairly crowded and so for stuff that are not purely a JS api, but a "glue function" used only by events, it's better to use the `evtTools` namespace (it won't "pollute" the generated GDJS documentation).